lib/oli/groups.ex

```diff
@@ -21,6 +21,12 @@ defmodule Oli.Groups do
   """
   def list_communities, do: Repo.all(Community)
 
+  defp filter_conditions(filter) do
```

Comment:
This seems like a random spot to put a private function - maybe move to bottom below public functions?